src/components/MlCreatePdfForm/MlCreatePdfForm.stories.tsx

```diff
@@ -78,7 +78,7 @@ const Template = () => {
-							const textBuffer = 1;
+							const textBuffer = 10;
```

Comment:
Please revert this change as it breaksthe PDF infobox layout, probably accidental.